app/controllers/appointment_summaries_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
   def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
+    OutputDocumentMailer.guidance_record(appointment_summary).deliver_later
```

Comment:
I wasn't happy with the method name here either, `guidance_record` was the only thing that seemed to fit at the time. I like `issue` much more.